app-creator/data/templates/templates.go

```diff
@@ -0,0 +1,50 @@
+package templates_data 
```

Comment:
No underscores in package names (https://golang.org/doc/effective_go.html#package-names).  Ideally, this should be the same as the file name, i.e., `templates`.